### DISCUSSION TITLE
Problem in mx.sym.Deconvolution()

### DISCUSSION THREAD

**Discussion by ducheng678** [apache/incubator-mxnet#12434] [Q&A]
This is the function in DOC:
mxnet.symbol.Deconvolution(data=None, weight=None, bias=None, kernel=_Null, stride=_Null, dilate=_Null, pad=_Null, adj=_Null, target_shape=_Null, num_filter=_Null, num_group=_Null, workspace=_Null, no_bias=_Null, cudnn_tune=_Null, cudnn_off=_Null, layout=_Null, name=None, attr=None, out=None, **kwargs)

This function has bias augment, but when I add bias in my code, I get following error:
mxnet.base.MXNetError: [14:57:26] c:\jenkins\workspace\mxnet-tag\mxnet\3rdparty\nnvm\src\core\symbolic.cc:72: Symbol.ComposeKeyword argument name bias not found. Candidate arguments: 	[0]data 	[1]weight

**Comment by anirudhacharya** [comment]
@mxnet-label-bot [Symbol, Question]

**Comment by Ishitori** [comment]
Yes, I confirm there is an issue. bias is defined in function signature, but if you try to actually put there a symbol you will get:

```
mxnet.base.MXNetError: [11:09:30] src/core/symbolic.cc:72: Symbol.ComposeKeyword argument name bias not found.
Candidate arguments:
	[0]data
	[1]weight
```

The minimal reproduce example is:

```
import mxnet as mx

deconv = mx.sym.Deconvolution(data=mx.sym.var('data'), kernel=(3, 3), num_filter=1,
                              bias=mx.symbol.var("bias", init=mx.initializer.Normal()))
```

@marcoabreu, please, retag
